typescript/tsconfig.go

```diff
@@ -41,6 +42,7 @@ func DefaultTsConfig() *TsConfig {
 			Target:            stringPointer("es2020"),
 			Module:            stringPointer("commonjs"),
 			OutDir:            stringPointer("out"),
+			RootDir:           stringPointer("src"),
```

Comment:
Can you change this so it defaults to `""` please?
The reason is because `tenderly actions init` command initializes the project without creating the `src` directory, so attempting to build a newly initialized project with `tenderly actions build` will lead to an error as it's trying to read from the wrong place.
This way, default behavior is preserved and users can always change the structure by editing the `tsconfig.json` file

```suggestion
			RootDir:           stringPointer(""),
```

Reply:
> Can you change this so it defaults to `""` please? The reason is because `tenderly actions init` command initializes the project without creating the `src` directory, so attempting to build a newly initialized project with `tenderly actions build` will lead to an error as it's trying to read from the wrong place. This way, default behavior is preserved and users can always change the structure by editing the `tsconfig.json` file